setup.sh

```diff
@@ -17,24 +17,16 @@ morgana_download="https://sourceforge.net/projects/morganaxproc-iiise/files/${mo
 
 if [ ! -f "${morgana}.zip" ]; then
     echo "Downloading Morgana XProc III SE ..."
-    curl -L -o "${morgana}.zip"  "${morgana_download}"
+    curl -L -o "${morgana}.zip" "${morgana_download}"
     unzip -qo "${morgana}.zip" -x __MACOSX/**
```

Comment:
Per our discussion, also mark it executable after unzip, minor change but important for first-time setup.

```suggestion
    curl -L -o "${morgana}.zip" "${morgana_download}"
    unzip -qo "${morgana}.zip" -x __MACOSX/**
    chmod +x "${morgana}/Morgana.sh"
```